test/mjsonwp/mjsonwp-specs.js

```diff
@@ -217,18 +217,32 @@ describe('MJSONWP', async () => {
         json: {id: 'baz', sessionId: 'lol', value: ['a']}
       });
 
+      // make sure adding the optional 'id' doesn't clobber a route where we
+      // have an actual required 'id'
       await request({
-        url: 'http://localhost:8181/wd/hub/session/foo/element/bar/value',
+        url: 'http://localhost:8181/wd/hub/session/foo/frame',
         method: 'POST',
         json: {id: 'baz'}
+      });
+    });
+
+    it('should properly handle variable payload for set value call', async () => {
+      await request({
+        url: 'http://localhost:8181/wd/hub/session/foo/element/bar/value',
+        method: 'POST',
+        json: {sessionId: 'lol'}
       }).should.eventually.be.rejectedWith("400");
 
-      // make sure adding the optional 'id' doesn't clobber a route where we
-      // have an actual required 'id'
       await request({
-        url: 'http://localhost:8181/wd/hub/session/foo/frame',
+        url: 'http://localhost:8181/wd/hub/session/foo/element/bar/value',
         method: 'POST',
-        json: {id: 'baz'}
+        json: {sessionId: 'lol', value: ['a'], text: 'bla'}
+      });
+
+      await request({
+        url: 'http://localhost:8181/wd/hub/session/foo/element/bar/value',
+        method: 'POST',
+        json: {sessionId: 'lol', text: 'bla'}
```

Comment:
this is precisely the right question to ask. because i want to make this change in BaseDriver, but I don't want to make a breaking change. That means that whatever change we make in BaseDriver, it needs to work with the existing function signatures of all the current drivers, including this FakeDriver.

the fact that the parameter is called `value` is inconsequential. basically we need to make sure that regardless of whether the user sends in a `value` or `text` JSON parameter, that it gets into the 2nd positional argument of the command.

in other words, if i send in a JSON request that looks like:

```json
{"value": "foo"}
```

the `args` returned should look like `[<elId>, "foo"]`

and if I send in:

```json
{"text": "foo"}
```

the `args` returned should _still_ look like `[<elId>, "foo"]`